src/funcx_common/task_storage/s3.py

```diff
@@ -13,6 +14,11 @@
 from .redis import ImplicitRedisStorage
 
 
+class Kind(Enum):
+    result = 1
+    payload = 2
```

Comment:
```suggestion
    result = "result"
    payload = "payload"
```

I see below the use of `Kind.name`. I recommend using `Enum.value`, and never (barring rare use-cases) `Enum.name`. The reason is that `Kind(Kind.result.value)` converts correctly, but `Kind(Kind.result.name)` only does if the two happen to match, which is not guaranteed. This becomes important when an enum is being specified externally as a string and needs to be converted.

As a result, generally `Enum.value` is the semantically significant one of these two values, and `Enum.name` is not.

Reply:
Done.

Reply:
I don't imagine the scenario you mentioned affecting this code, but it is simple enough to have it set to the name.

Reply:
> I don't imagine the scenario you mentioned affecting this code

Neither do I. But I want us to be internally consistent in how we use enums: the `value` part should always be the "externally significant" part of an enum.

In addition to the case I provided as an example, there are others. e.g. Given that enum member names need to be valid identifiers. So if you want an enum to contain `"foo-bar"`, that needs to come from the value part.

I see that you did end up making the change. Thanks for being willing to do that!